pages/linux/ncat.md

```diff
@@ -0,0 +1,12 @@
+# ncat
+
+> `cat` over networks.
+> Use the normal cat functionality over networks.
```

Comment:
Could these be combined into one line?

Reply:
Yes, I think the first line is redundant. And put cat within backticks.